packages/tasks/src/local-apps.ts

```diff
@@ -143,17 +147,29 @@ const snippetLocalAI = (model: ModelData, filepath?: string): LocalAppSnippet[]
 };
 
 const snippetVllm = (model: ModelData): LocalAppSnippet[] => {
-	const runCommand = [
+	const runCommandInstruct = [
 		"# Call the server using curl:",
-		`curl -X POST "http://localhost:8000/v1/chat/completions" \\ `,
-		`	-H "Content-Type: application/json" \\ `,
+		`curl -X POST "http://localhost:8000/v1/chat/completions" \\`,
+		`	-H "Content-Type: application/json" \\`,
 		`	--data '{`,
 		`		"model": "${model.id}",`,
 		`		"messages": [`,
 		`			{"role": "user", "content": "Hello!"}`,
 		`		]`,
 		`	}'`,
 	];
+	const runCommandNonInstruct = [
+		"# Call the server using curl:",
+		`curl -X POST "http://localhost:8000/v1/completions" \\`,
+		`	-H "Content-Type: application/json" \\`,
+		`	--data '{`,
+		`		"model": "${model.id}",`,
+		`		"prompt": "Once upon a time ",`,
```

Comment:
```suggestion
		`		"prompt": "Once upon a time",`,
```

Remove trailing space in the prompt, adding a space almost always results in weird generations.